src/calculateRentalCost.js

```diff
@@ -4,7 +4,17 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const rentalCost = days * 40;
+
+  if (days >= 7) {
+    return rentalCost - 50;
+  }
+
+  if (days >= 3) {
+    return rentalCost - 20;
+  }
+
+  return rentalCost;
```

Comment:
You can remove the 'else' structure by using 'return' statements effectively. If a condition is met and the function returns, the subsequent conditions do not need to be wrapped in an 'else' block.